calico-cloud/reference/resources/runtimesecurity.mdx

```diff
@@ -0,0 +1,109 @@
+
+### RuntimeReporterDaemonSetPodTemplateSpec
+
+| Field    | Description                                        | Accepted Values | Schema                           |
+| -------- | -------------------------------------------------- | --------------- | -------------------------------- |
+| spec     | Spec is the runtime reporter DaemonSet's PodSpec.  |                 | [RuntimeReporterDaemonSetPodSpec](#runtimereporterdaemonsetpodspec)  |
+
+### RuntimeReporterDaemonSetPodSpec 
+
+If any of the fields below are specified, they will override the default runtime reporter DaemonSet values for these fields. Otherwise they will use the default values
+
+| Field        | Description                                                                               | Accepted Values | Schema                                                                                                         | Default |
+| --------     | ----------------------------------------------------------------------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------- | ------- |
+| containers   | Containers is a list of runtime reporter DaemonSet containers.                            |                 | [RuntimeReporterDaemonSetContainer](#runtimereporterdaemonsetcontainer)                                        |         |                                      |         |
+| affinity     | Affinity is a group of affinity scheduling rules for the runtime reporter DaemonSet pods. |                 | [corev1.Affinity](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#affinity-v1-core)       |         |
+| nodeSelector | NodeSelector is the runtime reporter pod's scheduling constraints.                        |                 | Map                                                                                                            |         |
+| tolerations  | Tolerations is the runtime reporter pod's tolerations.                                    |                 | [corev1.Tolerations](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#toleration-v1-core)  |         |
+
+### RuntimeReporterDaemonSetContainer
+
+| Field    | Description                                                                        | Accepted Values | Schema                      |
+| -------- | ---------------------------------------------------------------------------------- |---------------- | --------------------------  |
+| name     | Name of the runtime reporter daemonset container                                   |                 | [RuntimeReportDaemonSetSpec](#runtimereporterdaemonsetspec)  |
+| resource | Customization of limits and requests for compute resources such as cpu and memory. |                 | [corev1.ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#resourcerequirements-v1-core) |
```

Comment:
Not your fault, but I think this is a horrible presentation.  If we are going to document RuntimeReporterDaemonSet at all, I think we must do so by presenting a flat list of all of the fields that it allows to be overwritten, i.e. something like
```
- Spec.Template.Spec.Containers[*].Resources
- Spec.Template.Spec.Affinity
- ...
```
I've tried reading the relevant code to work out what that complete flat list would be, but the code is so horrible that I cannot.  I think our choices are:
1. Don't document this field at all (as already suggested in an earlier comment).
2. Go back to whoever developed this work and ask them to provide that flat list.

Reply:
I would prefer to go with option 1.
We can always iterate later as needed.

For option 2, @jaderhs, do you know if other components that also support pod affinity etc have had their docs updated? If so we should follow the same format. If not, that’s a strong case for option 1 (we can always log a task to address that separately).

Reply:
ImageAssurance & Compliance benchmarking has documented these fields
- https://docs.tigera.io/calico-cloud/reference/installation/ia-api#image-assurance.operator.tigera.io/v1.CrawdadDaemonSet
- https://docs.tigera.io/calico-cloud/reference/installation/api#operator.tigera.io/v1.ComplianceBenchmarkerDaemonSet

But I am more than happy to remove references to these fields from the Docs altogether